song-helper/fake_server.py

```diff
@@ -0,0 +1,31 @@
+from .utils import SpotifyUtils
+from .predict import PredictMood
+
+
+class FakeServer:
+    """
+    This fake implements the requests from server-side interface.
+    This interface is also used by the real implementation
+    """
+    def __init__(self, spt: SpotifyUtils = None, pt: PredictMood = None) -> None:
+        self.spt = spt
+        self.pt = pt
+
```

Comment:
Maybe We need more spaces between "def" like in start.py?

Reply:
No, linter shows 0 errors

Reply:
Ok, them